treatment-service/controller/treatment.go

```diff
@@ -201,41 +201,7 @@ func (t TreatmentController) FetchTreatment(w http.ResponseWriter, r *http.Reque
 	Ok(w, response, &requestId)
 }
 
-func (t TreatmentController) logFetchTreatmentMetrics(
```

Comment:
This method has been refactored as a metric service method so that it can be reused by xp's plugin.